hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -634,9 +629,9 @@ public void testInvalidPrefixAcl() throws IOException {
         .build();
 
     // add acl with invalid prefix name
-    exception.expect(OMException.class);
-    exception.expectMessage("Invalid prefix name");
-    prefixManager.addAcl(ozInvalidPrefix, ozAcl1);
+    OMException ex = assertThrows(OMException.class, () ->
```

Comment:
@adoroszlai This is different from #4257, so there must be some error introduced on the master.

Reply:
HDDS-8157 is created to track this issue, and HDDS-6964(#3653) may be the cause.